views/index.html

```diff
@@ -1,31 +1,116 @@
 <!DOCTYPE html>
 <html>
-  <head>
+<head>
     <meta charset="utf-8">
-    <title>{{ title }}</title>
+    <title>{% if title %}{{ title }} : {% endif %}Developers' Guild</title>
```

Comment:
As said below, we don't need to check to see if `title` is valid because there is only one instance where we load this page and it's always valid there.